rsa.js

```diff
@@ -20,26 +20,72 @@ function sanitize(key) {
   return key.replace(new RegExp('\n|\r', 'g'), '');
 }
 
+function removeEndingEquals(blob) {
+  if (blob.endsWith('==')) {
+    return blob.slice(0,-2);
+  } else if (blob.endsWith('=')) {
+    return blob.slice(0,-1);
+  }
+  return blob;
+}
+
+function illegalBase64(blob) {
```

Comment:
I agree that it makes the code too complex only to slightly improve error reporting to the user. I'll remove it !